ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -239,4 +239,94 @@ public static synchronized String getSchemaDefinitionString(Schema schema) {
     return stringBuilder.toString();
   }
 
+  public static String getSQLTypeName(Schema schema) {
+    switch (schema.type()) {
```

Comment:
we seem to be alot of `switch(schema.type())` in this class. IMO it would be better to have a class that represents the things we need to convert to. Register instances of the class in a map. Then in the methods where we are doing all the `switch(schema.type())` we do, for example: `map.get(schema.type()).getJavaType()` etc

Reply:
good point, this change will affect other formats along with avro too. Will refactor this class along with adding tests in another PR.

Reply:
How are we tracking all the work which is being postponed? We really should file issues for each.